packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -96,6 +96,117 @@ It requires no configuration, but is included in your app when you use a Toolkit
 AriaAnnouncer is ever required, only the outermost instance of ToolkitProvider will instantiate the AriaAnnouncer. If you use
 nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
 
+## useDensity
+
+You can use `useDensity` hook to access the currently active density of your app in the provided scope.
+
+### API
+
+```js
+import { useDensity } from "@jpmorganchase/uitk-core";
+```
+
+### Example use
+
+Use density hook returns the string value of the currently set density in scope. You can optionally override the value bu passing your own value as the first parameter to the function.
```

Comment:
The 'useDensity' hook...

Reply:
typo '...optionally override the value **by** passing...'

Reply:
This comment applies to the whole page - it would be good to list the possible values that each hook could return i.e. 'The useDensity hook returns the string value of the currently set density in scope. The possible values are ...'

Reply:
Good point